qiskit_metal/renderers/renderer_gds/gds_renderer.py

```diff
@@ -2343,13 +2343,13 @@ class to a series of GDSII format on the input pd.Series.
                                                use_width,
                                                layer=qgeometry_element.layer,
                                                max_points=max_points,
-                                               datatype=11)
+                                               datatype=qgeometry_element.sublayer+1)
```

Comment:
This is temporary. I need to properly check here whether it is `None` or not, and make it 11 as a default. 
Otherwise, I am losing the ability to have Flexpaths and Polygon on the same datatype